sn_networking/src/lib.rs

```diff
@@ -288,9 +301,9 @@ impl Network {
                     }
                 }
                 Err(Error::RecordNotEnoughCopies(returned_record)) => {
-                    debug!("Not enough copies found yet...");
+                    debug!("Not enough copies found yet for {pretty_key:?}");
                     // Only return when completed all attempts
-                    if verification_attempts >= total_attempts {
+                    if verification_attempts >= total_attempts && matches!(quorum, GetQuorum::One) {
```

Comment:
is the `match` really necessary ?
the original idea of this `return ok branch in case of not having enough copy` is to allow a consensused return
as long as the fetched copy is the only one or matches the expected one, even when the quorum is Majority or All.

Reply:
Without this, we return OK for all chunks stored even when that's not true